api/get.js

```diff
@@ -126,8 +139,13 @@ Api.prototype = {
         var lookup = {
             search: '?' + query,
             query: query,
-            path: req.url
+            binding: req.url
```

Comment:
This should be "path". it represents the current path passed by the url. It has nothing to do with the binding variable used on config.

The lookup object is a temp proxy to be used later when checking the config.url
The code will look for {{variable}} in the url path and replaced it by the content of lookup[variable], i.e., ```http://google.com/{{path}}``` will become ```http://google/q```